clang/lib/Sema/SPIRVBuiltins.td

```diff
@@ -929,13 +929,13 @@ foreach name = ["GroupBroadcast"] in {
   }
 }
 
-foreach name = ["GroupIAdd", "GroupNonUniformIMul", "GroupNonUniformBitwiseOr",
-                "GroupNonUniformBitwiseXor", "GroupNonUniformBitwiseAnd"] in {
+foreach name = ["GroupIAdd", "GroupIMulKHR", "GroupBitwiseOrKHR",
+                "GroupBitwiseXorKHR", "GroupBitwiseAndKHR"] in {
```

Comment:
https://github.com/KhronosGroup/SPIRV-Registry/pull/138/files?short_path=771b867#diff-771b867fa2b28c0a331ac547cfe35642695f0d4da40653e89278a2f188eb2eb4 doesn't seem to say that it removes old instructions, it says it adds new ones. So, I wonder why old ones are not preserved?

Reply:
Currently we're having mixed implementations in functional.hpp (uniform and non-uniform) but we want to unify it and make all of them uniform.